clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -308,6 +308,15 @@ def err_target_unsupported_type_for_abi
     : Error<"%0 requires %1 type support, but ABI '%2' does not support it">;
 }
 
+def err_sycl_illegal_virtual_call
+    : Error<"kernel '%0' performs virtual function call and they are illegal "
+            "per the core SYCL 2020 specification. To enable their support, "
+            "submit a kernel using 'calls_indirectly' property, see "
+            "sycl_ext_oneapi_virtual_functions extension for more information">,
```

Comment:
```suggestion
            "the sycl_ext_oneapi_virtual_functions extension for more information">,
```

Reply:
Applied in 120fbc38ca4411b1bc8c3a4d986adc90dd5eafd4